docs/guidelines/comments.rst

```diff
@@ -0,0 +1,125 @@
+Comments
+========
+
+.. note::
+
+    If peppering your code with lots of comments is good, then having zillions of comments in your code must be great,
+    right? Not quite. Comments in code are a double edged sword, use them to describe what's being done.
+
+
+It doesn't make sense to comment every step you do in your program. Comments should discuss why something is done,
+its purpose and its goal. The code already shows how it is done, so commenting on this is redundant.
+
+Section Comments
+----------------
+
+The first separation style is *section comment*. This uses the full 120 width span ideally placed to separate large
+chunks of logic::
+
+    // #####################################################################################################################
```

Comment:
tbh I'd love to see these removed.
main reason we use it is to separate different blocks in the scss file. we don't really do that in js anyway, since we separate different functionalities into different files. why not do the same in scss?